wordpress-mobile/WordPress-iOS
# Use Reader V2 posts endpoint for topics

Fixes #14930 

### To test

1. Open Discover
2. Tap in any topic under "Topics you might like"
3. Check that the content appears correctly

Pull to refresh, scroll, etc. :)
